qiskit/extensions/quantum_initializer/ucg.py

```diff
@@ -121,7 +121,7 @@ def _dec_ucg(self):
         circuit = QuantumCircuit(q)
         # If there is no control, we use the ZYZ decomposition
         if not q_controls:
-            theta, phi, lamb = euler_angles_1q(self.params[0])
+            theta, phi, lamb, _ = OneQubitEulerDecomposer._params_u3(self.params[0])
```

Comment:
Yep, I had that before but since its just using this static method I did it directly so it wouldn't require initializing an object. Can change it back though.

Reply:
I think that would be cleaner. Initialization should not be costly. Also there is a similar line in ` qiskit/extensions/unitary.py`.